src/shared/GitHub/GitHubAuthentication.cs

```diff
@@ -231,29 +229,11 @@ public async Task<OAuth2TokenResult> GetOAuthTokenAsync(Uri targetUri, IEnumerab
 
         private bool TryFindHelperExecutablePath(out string path)
         {
-            string helperName = GitHubConstants.AuthHelperName;
-
-            if (PlatformUtils.IsWindows())
-            {
-                helperName += ".exe";
-            }
-
-            string executableDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-            path = Path.Combine(executableDirectory, helperName);
-            if (!Context.FileSystem.FileExists(path))
-            {
-                Context.Trace.WriteLine($"Did not find helper '{helperName}' in '{executableDirectory}'");
-
-                // We currently only have a helper on Windows. If we failed to find the helper we should warn the user.
-                if (PlatformUtils.IsWindows())
```

Comment:
I guess that losing this part isn't dramatic, eh?

Reply:
Not really, as it points to an installation issue which is only resolved by reinstalling. There is a fallback for scenarios when you have terminal access.

Reply:
Okay!